pipeline/package.json

```diff
@@ -18,12 +18,13 @@
     "typescript": "~3.7.2"
   },
   "dependencies": {
-    "@aws-cdk/aws-codebuild": "^1.57.0",
-    "@aws-cdk/aws-codepipeline": "^1.57.0",
-    "@aws-cdk/aws-codepipeline-actions": "^1.57.0",
-    "@aws-cdk/aws-s3": "^1.57.0",
-    "@aws-cdk/aws-ssm": "^1.57.0",
-    "@aws-cdk/core": "1.57.0",
-    "source-map-support": "^0.5.16"
+    "@aws-cdk/assets": "^1.105.0",
+    "@aws-cdk/aws-codebuild": "^1.105.0",
+    "@aws-cdk/aws-codepipeline": "^1.105.0",
+    "@aws-cdk/aws-codepipeline-actions": "^1.105.0",
+    "@aws-cdk/aws-s3": "^1.105.0",
+    "@aws-cdk/aws-ssm": "^1.105.0",
```

Comment:
I think we don't need aws-ssm anymore.

Reply:
I agree and I changed on my repo 
Can you change on your repo? Otherwise, I will need to do a pull request again.

Reply:
@jbernalvallejo 
I have submitted a commit with the changes and I believe that it also reflects this PR. Let me know if you can see it.